pkg/cli/client_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build extended
-// +build extended
+//go:build extended || needprivileges
```

Comment:
Why need this change?

Reply:
One of the tests with elevated privileges needed this before due to imports, but another change along the way must have fixed it. Removed it in the latest commit